cfpb/cfgov-refresh
# PFC model update

This PR sets the stage for our college-cost decoder, an open version of
the dynamic disclosures tool built to help students at for-profit colleges.

Its intent is to establish our model changes in the cfgov database so that we can
iterate on importing and using new data points and implementing a new tool design.

### Included
- Three new page models to accommodate the open tool, a guide-quiz format and static legacy pages.
- New fields on the `school` model to hold data we will collect from the College Scorecard api.
- Blocks for the quizzes.
- Initial templates and front-end resources for the new pages.

### Not included
- Code to harvest the new data from Scorecard
- Final versions of templates and front-end code

### Testing
- Import the branch and run the new migration against a recent prod db snapshot:
```bash
./cfgov/manage.py migrate
```
- Add a child page to the PFC placeholder (http://localhost:8000/admin/pages/12466/) and choose **Student loan quiz page** 
- In the admin, note the Guided Quiz block available on the page, and try adding one. You can see a populated version of the guided quiz internally at [DEV5]/paying-for-college/choose-a-student-loan/
- You can also try adding a College costs page or a Repaying student debt page. They should render, but they are not fully functional.
